src/common/eth-providers/execution-provider/execution-provider.module.ts

```diff
@@ -3,14 +3,15 @@ import { FallbackProviderModule } from '@lido-nestjs/execution';
 import { PrometheusService, RequestStatus } from 'common/prometheus';
 import { ConfigService } from 'common/config';
 import { ExecutionProviderService } from './execution-provider.service';
+import { NonEmptyArray } from '@lido-nestjs/execution/dist/interfaces/non-empty-array';
```

Comment:
Fill free to contribute to `@lido-nestjs` module to export used interface.

Reply:
used another source